<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AndroidSettingsWidget</class>
 <widget class="QWidget" name="AndroidSettingsWidget">
  <property name="windowTitle">
   <string>Android Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="androidSettingsGroupBox">
     <property name="title">
      <string>Android Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="Utils::PathChooser" name="SDKLocationPathChooser" native="true"/>
      </item>
      <item row="1" column="6">
       <widget class="QToolButton" name="downloadSDKToolButton">
        <property name="toolTip">
         <string>Open Android SDK download URL in the system's browser.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" rowspan="3" colspan="4">
       <layout class="QVBoxLayout" name="ndkListControlsLayout">
        <item>
         <widget class="QPushButton" name="addCustomNdkButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Add the selected custom NDK. The toolchains and debuggers will be created automatically.</string>
          </property>
          <property name="text">
           <string>Add...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeCustomNdkButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Remove the selected NDK if it has been added manually.</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="makeDefaultNdkButton">
          <property name="toolTip">
           <string>Force a specific NDK installation to be used by all Android kits.&lt;br/&gt;Note that the forced NDK might not be compatible with all registered Qt versions.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="7">
       <widget class="Utils::DetailsWidget" name="androidDetailsWidget" native="true"/>
      </item>
      <item row="3" column="1" rowspan="3">
       <widget class="QListWidget" name="ndkListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="OpenJDKLocationLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>JDK location:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="OpenJDKLocationPathChooser" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="downloadOpenJDKToolButton">
        <property name="toolTip">
         <string>Open JDK download URL in the system's browser.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="SDKLocationLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Android SDK location:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="4">
       <widget class="QPushButton" name="sdkToolsAutoDownloadButton">
        <property name="text">
         <string>Set Up SDK</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="4">
       <widget class="QPushButton" name="sdkManagerToolButton">
        <property name="text">
         <string>SDK Manager</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ndkComboBoxLabel">
        <property name="text">
         <string>Android NDK list:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QToolButton" name="downloadNDKToolButton">
        <property name="toolTip">
         <string>Open Android NDK download URL in the system's browser.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="7">
       <widget class="QCheckBox" name="CreateKitCheckBox">
        <property name="text">
         <string>Automatically create kits for Android tool chains</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="androidOpenSSLSettingsGroupBox">
     <property name="title">
      <string>Android OpenSSL settings (Optional)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>OpenSSL binaries location:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="Utils::DetailsWidget" name="openSslDetailsWidget" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="downloadOpenSSLPrebuiltLibs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Download OpenSSL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="Utils::PathChooser" name="openSslPathChooser" native="true">
        <property name="toolTip">
         <string>Select the path of the prebuilt OpenSSL binaries.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>Utils::DetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">utils/detailswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
